openassessment/xblock/openassessmentblock.py

```diff
@@ -751,6 +751,18 @@ def in_studio_preview(self):
         # student ID, so we can't rely on that)
         return self.scope_ids.user_id is None
 
+    @property
+    def in_studio(self):
```

Comment:
I feel uneasy about this for some reason, and I think we should instead be explicit and just call this "has_no_real_user" or something like that. It's a blurry, kinda backwards way of checking if we're in studio. It works, but I think we'd be safer just calling this what it is, and plus, it makes sense down the line too. Why do we care about not submitting in studio? Because there's no real user. Cut out the middle step and hide the button if there's no user. Does that make sense? I'm kinda rambling.